patches/chrome-browser-chrome_browser_interface_binders_webui_parts_desktop.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/chrome_browser_interface_binders_webui_parts_desktop.cc b/chrome/browser/chrome_browser_interface_binders_webui_parts_desktop.cc
+index e3bef9cb9d069a146c92fbfdfd4439a3bd45257c..c0323a0cdeb9500df2b1a614b2239cd658c8e794 100644
+--- a/chrome/browser/chrome_browser_interface_binders_webui_parts_desktop.cc
++++ b/chrome/browser/chrome_browser_interface_binders_webui_parts_desktop.cc
+@@ -574,6 +574,7 @@ void PopulateChromeWebUIFrameInterfaceBrokersTrustedPartsDesktop(
+ 
+   registry
+       .ForWebUI<settings::SettingsUI>()
++          BRAVE_POPULATE_CHROME_WEBUI_FRAME_INTERFACE_BROKERS_TRUSTED_PARTS_DESKTOP
```

Comment:
I don't think this is an ideal name for this patch hook - it sounds like it will let you inject binders into any desktop WebUI but it will only let you add them to the `SettingsUI`

Reply:
That's our standard patch name pattern: `BRAVE_` + the function name into which the patch goes (in this case `void PopulateChromeWebUIFrameInterfaceBrokersTrustedPartsDesktop(...)`). We can add more details at the end, but it's already pretty long :D